server/api/auth/resetPassword.ts

```diff
@@ -0,0 +1,37 @@
+import { Express, Response, Request } from 'express'
+import { ApiEndPoint } from '@common/api/endpoint'
+import { validEmail } from '@common/userUtils'
+import { Objects } from '@core/utils'
+import { UserController } from '@server/controller'
+import { sendErr, serverUrl } from '@server/utils/requests'
+
+export const AuthResetPassword = {
+  init: (express: Express): void => {
+    express.post(ApiEndPoint.Auth.ResetPassword.create(), async (req: Request, res: Response) => {
+      try {
+        const { email } = req.body
+
+        if (Objects.isEmpty(email)) {
+          res.send({ error: 'login.emptyEmail' })
+        } else if (!validEmail({ email })) {
+          res.send({ error: 'login.invalidEmail' })
+        } else {
```

Comment:
see next comment

this can be written probably like

```
        if (Objects.isEmpty(email)) res.send({ error: 'login.emptyEmail' })
        if (!validEmail({ email })) res.send({ error: 'login.invalidEmail' })
```
to remove some nesting